uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/MicrosoftPromptHandler.java

```diff
@@ -0,0 +1,65 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.interaction;
+
+public class MicrosoftPromptHandler extends AbstractPromptHandler {
```

Comment:
nit: rename to `AzureActiveDirectoryPromptHandler` or `AadPromptHandler`

Reply:
+1 on `AadPromptHandler`

Reply:
Is it AAD v1 or v2?  If v2 i'd mirror what we do relative to the strategy for v2... which i think is MIcrosoftSTS or something like that.

Reply:
It is v2, I would personally want to stick to `AadPromptHandler` as we will need different prompt handler for different policies and identity providers (once we get to writing those tests). Will probably need a different one for ADFS as well.

Reply:
Addressed by renaming to `AadPromptHandler`